test/Microsoft.AspNet.Mvc.FunctionalTests/compiler/resources/RazorWebSite.UrlResolution.Index.html

```diff
@@ -9,12 +9,12 @@
     <link href="/Person/John Doe/CSS" rel="stylesheet">
     <video poster="/SomeUrl" src="/SomeUrl/video" />
     <audio src="/SomeUrl">
-        <source src="/Person">
+        <source src="/Person" srcset="/Person">
         <track src="/email~/SomeUrl">
     </audio>
     <embed src="/email@dyanmicUrl">
     <iframe src="/SomeUrl" />
-    <img src="/~/SomeUrl">
+    <img src="/~/SomeUrl" srcset="/~/SomeUrl">
```

Comment:
:-1: 
It's a functional test. We're purposefully doing things that user code wouldn't normally do; this file tries to do everything known to man for variations of `~/` :smile:.